api/static/js/details.js

```diff
@@ -36,7 +36,7 @@ function fetchInfo(con_name) {
     }
 
     $.ajax({
-        url: '/contract',
+        url: '/agreement/all',
```

Comment:
How does this work? I see this code in the GET API:
		id := pathVars["id"]

		// we don't support getting just one yet
		if id != "" {
			w.WriteHeader(http.StatusBadRequest)
			return
		}
Does this pick up the /all and return an error?